### DISCUSSION TITLE
拖入整合包时候下载安装Forge失败，开梯子也没啥用，请有懂的大佬帮瞅瞅吗

### DISCUSSION THREAD

**Discussion by shixiaogaoya** [Meloong-Git/PCL#4331] [提问] [answered]
游戏安装失败
→ 下载 Forge 47.3.5失败
→ 安装 Forge（方式 A）失败
→ 安装新 Forge 版本失败
→ 运行 Forge 安装器失败
→ Forge 安装器出错，日志结束部分为：
	... 5 more
Caused by: java.nio.charset.UnsupportedCharsetException: GBK
	at java.nio.charset.Charset.forName(Unknown Source)
	at sun.nio.fs.Util.<clinit>(Unknown Source)
	... 18 more
   在 PCL.ModDownloadLib._Closure$__35-0._Lambda$__7(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__13-0._Lambda$__0()
   在 PCL.ModDownloadLib._Closure$__35-0._Lambda$__7(LoaderTask`2 Task)
   在 PCL.ModDownloadLib.ForgelikeInjector(String Target, LoaderTask`2 Task, String McFolder, Boolean UseJavaWrapper, Boolean IsNeoForge)
   在 PCL.ModDownloadLib._Closure$__35-0._Lambda$__7(LoaderTask`2 Task)

**Comment by allMagicNB** [comment]
上传日志

**Reply by shixiaogaoya** [reply]
[Log4.txt](https://github.com/user-attachments/files/16360692/Log4.txt)
[Log5.txt](https://github.com/user-attachments/files/16360693/Log5.txt)
[Log2.txt](https://github.com/user-attachments/files/16360694/Log2.txt)
[Log3.txt](https://github.com/user-attachments/files/16360695/Log3.txt)

**Comment by MoYuan-CN** [comment]
GBK 编码不被支持……
上传你的 PCL 日志以分析问题

**Reply by shixiaogaoya** [reply]
很抱歉再次打扰你，因为我发现还是行不通，麻烦大佬再帮我看一下，（跪谢~~~）
[Log1.txt](https://github.com/user-attachments/files/16415599/Log1.txt)

**Comment by shixiaogaoya** [comment]
[Log4.txt](https://github.com/user-attachments/files/16360335/Log4.txt)
[Log5.txt](https://github.com/user-attachments/files/16360336/Log5.txt)
[Log2.txt](https://github.com/user-attachments/files/16360337/Log2.txt)
[Log3.txt](https://github.com/user-attachments/files/16360338/Log3.txt)

**Comment by hejiehao** [comment]
把 JAVA_TOOL_OPTIONS 从环境变量里删掉

**Reply by shixiaogaoya** [reply]
好的我待会试试，谢谢回复😊

**Reply by shixiaogaoya** [reply]
很抱歉再次打扰你，因为我发现还是行不通，麻烦大佬再帮我看一下，（跪谢~~~）
[Log1.txt](https://github.com/user-attachments/files/16415567/Log1.txt)

**Comment by MoYuan-CN** [comment]
新的日志中报错是一样的，怀疑系统环境问题？

**Comment by LTCatt** [comment, marked as answer]
转到 #4374
